docs/source/_sidebar.yaml

```diff
@@ -4,22 +4,39 @@ switcher:
     - label: v0
       latest: true
       href: ./
+defaultOpenDepth: 2
 items:
-  - label: "Welcome"
+  - label: "Overview"
     href: "."
   - label: "Quickstart"
```

Comment:
Should we rename this label "Get Started"? The title for the page is that and it seems to be the pattern for other quickstart guides in our docs

Reply:
Yes you're right! I'll log this with an existing ticket where we tackle the contents of the page 👍🏽